gremlin-driver/src/test/java/org/apache/tinkerpop/gremlin/driver/TestWSGremlinInitializer.java

```diff
@@ -135,9 +135,15 @@ protected void decode(final ChannelHandlerContext ctx, final BinaryWebSocketFram
             } else if (msg.getRequestId().equals(RESPONSE_CONTAINS_SERVER_ERROR_REQUEST_ID)) {
                 Thread.sleep(1000);
                 ctx.channel().writeAndFlush(new CloseWebSocketFrame());
-            }
-            else if (msg.getRequestId().equals(USER_AGENT_REQUEST_ID)) {
+            } else if (msg.getRequestId().equals(USER_AGENT_REQUEST_ID)) {
                 ctx.channel().writeAndFlush(new TextWebSocketFrame(returnSimpleStringResponse(USER_AGENT_REQUEST_ID, userAgent)));
+            } else {
```

Comment:
I'm not opposed to this case being added but it is functionally the same as sending a request with `SINGLE_VERTEX_REQUEST_ID` as in ```
client.submit("1",
          RequestOptions.build().overrideRequestId(TestWSGremlinInitializer.SINGLE_VERTEX_REQUEST_ID).create())
```